src/webpack.tooling.js

```diff
@@ -119,12 +119,6 @@ export const minify = () => ({
     minimizer: [
       new TerserPlugin({
         sourceMap: true,
-        cache: true,
```

Comment:
Is dropping cache as well intentional?

Reply:
I read Alec's description as implying it'll still be enabled anyway, even w/o this line

Reply:
Yep, just because true is the default value so no need to configure it explicitly